index.html

```diff
@@ -242,7 +242,6 @@ <h1 class="Header">About Me</h1>
         <hr class="AboutLine2">
```

Comment:
- [ ] I noticed a small issue with your contact form. It seems that there's a bug **causing horizontal scrolling**. 🎯 This could be due to a `width` problem. To help you better understand, I've recorded a video demonstrating the issue:

   [6_14_2023_12-11-46 AM.webm](https://github.com/bhushan354/Portfolio_Project/assets/107952724/3199118b-925f-416e-9cdc-55f82373b61c)